cmd/gb/env.go

```diff
@@ -13,37 +11,14 @@ func init() {
 
 var EnvCmd = &cmd.Command{
 	Name:      "env",
-	UsageLine: `env`,
+	UsageLine: `env [var ...]`,
```

Comment:
Thank you. I cound’t find a decent way to parse arguments (only flags) using getopt. Implemented it simlar to the https://github.com/golang/go/blob/master/src/cmd/go/env.go#L16